### DISCUSSION TITLE
Wave: How to forward to specific path if entered path is not exists.

### DISCUSSION THREAD

**Discussion by italanchan** [h2oai/wave#1829] [Q&A]
Hi, 

Wave: How to forward to specific path if entered path or router is not exists.

**Comment by mturoci** [comment]
Hi @italanchan. Can you add more details? I am not sure what you mean.

**Reply by italanchan** [reply]
Refer to https://wave.h2o.ai/docs/examples/hello-world, 

The page is http://localhost:10101/demo

When user direct the url in browser, e.g. http://localhost:10101/not_exist_demo

As the page is not available, we want it auto redirect to the default page (http://localhost:10101/demo) or customs 404 page.

**Reply by mturoci** [reply]
Got it now. Wave doesn't support this which means you can currently only achieve the desired behavior by setting up a proxy (like nginx etc.) to forward the traffic the way you want/need. We might support custom 404s in the future though.
